[2024-10-08T14:30:45.706+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-08T14:30:45.775+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_dag.preprocess_data manual__2024-10-08T14:30:01.235157+00:00 [queued]>
[2024-10-08T14:30:45.815+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_dag.preprocess_data manual__2024-10-08T14:30:01.235157+00:00 [queued]>
[2024-10-08T14:30:45.816+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-08T14:30:45.850+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): preprocess_data> on 2024-10-08 14:30:01.235157+00:00
[2024-10-08T14:30:45.896+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=94) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-08T14:30:45.903+0000] {standard_task_runner.py:72} INFO - Started process 101 to run task
[2024-10-08T14:30:45.904+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline_dag', 'preprocess_data', 'manual__2024-10-08T14:30:01.235157+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmptkx6gfx3']
[2024-10-08T14:30:45.907+0000] {standard_task_runner.py:105} INFO - Job 30: Subtask preprocess_data
[2024-10-08T14:30:46.145+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline_dag.preprocess_data manual__2024-10-08T14:30:01.235157+00:00 [running]> on host d2aa38be11ee
[2024-10-08T14:30:46.269+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline_dag' AIRFLOW_CTX_TASK_ID='preprocess_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-08T14:30:01.235157+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-08T14:30:01.235157+00:00'
[2024-10-08T14:30:46.274+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-08T14:30:46.297+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-08T14:30:46.298+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/scripts/runtime.py /opt/***/scripts/preprocess.py']
[2024-10-08T14:30:46.315+0000] {subprocess.py:86} INFO - Output:
[2024-10-08T14:30:48.509+0000] {subprocess.py:93} INFO - Error: ../data/reddit_dataset_2011.csv does not exist. Please check the file path.
[2024-10-08T14:30:49.564+0000] {subprocess.py:93} INFO - 
[2024-10-08T14:30:49.585+0000] {subprocess.py:93} INFO - Starting script: /opt/***/scripts/preprocess.py
[2024-10-08T14:30:49.588+0000] {subprocess.py:93} INFO - Finished script: /opt/***/scripts/preprocess.py
[2024-10-08T14:30:49.589+0000] {subprocess.py:93} INFO - Execution time: 3.02 seconds
[2024-10-08T14:30:49.591+0000] {subprocess.py:93} INFO - Average CPU usage: 100.30%
[2024-10-08T14:30:49.592+0000] {subprocess.py:93} INFO - Maximum RAM usage: 99.69 MB
[2024-10-08T14:30:49.593+0000] {subprocess.py:93} INFO - 
[2024-10-08T14:30:49.594+0000] {subprocess.py:93} INFO - ===== Summary =====
[2024-10-08T14:30:49.596+0000] {subprocess.py:93} INFO - Script: /opt/***/scripts/preprocess.py
[2024-10-08T14:30:49.597+0000] {subprocess.py:93} INFO -   Execution time: 3.02 seconds
[2024-10-08T14:30:49.597+0000] {subprocess.py:93} INFO -   Average CPU usage: 100.30%
[2024-10-08T14:30:49.598+0000] {subprocess.py:93} INFO -   Maximum RAM usage: 99.69 MB
[2024-10-08T14:30:49.598+0000] {subprocess.py:93} INFO - 
[2024-10-08T14:30:49.598+0000] {subprocess.py:93} INFO - Total execution time for all scripts: 3.02 seconds
[2024-10-08T14:30:49.610+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-08T14:30:49.862+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-08T14:30:49.872+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline_dag, task_id=preprocess_data, run_id=manual__2024-10-08T14:30:01.235157+00:00, execution_date=20241008T143001, start_date=20241008T143045, end_date=20241008T143049
[2024-10-08T14:30:49.991+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-08T14:30:50.067+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-08T14:30:50.069+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
